scripts/reset-test-users.sh

```diff
@@ -2,27 +2,34 @@
 
 function resetTestUsers() {
 
+source ./scripts/reset-account-intervention.sh
```

Comment:
think this should be in 'reset-test-data.sh' at the top with the other two

Reply:
done